### ISSUE TITLE
Inline images

### ISSUE THREAD

**Issue by pejuko** [bradpauly/griddler-mailgun#8]
Does griddle-mailgun support inline images? Mailgun creates content-id-map which I don't see when I use griddler-mailgun. I need to link inline images referenced by cid to attachment but the Content-ID is not presented between headers.

**Comment by bradpauly** [comment]
It doesn't right now. It supports the attributes defined by https://github.com/thoughtbot/griddler

I'm pretty sure you could get that out of the headers. Did you try that?

**Comment by pejuko** [comment]
Yes I did, but Content-ID is missing in headers because mailgun deletes them from headers and they are creating their own param content-id-map :-/ I've added support for this so you can get content-id-map as hash from params now.

**Comment by pejuko** [comment]
Hm, I extended the param but this is probably wrong as it is private. Any ideas how to get content-id-map into email?

**Comment by bradpauly** [comment]
I'll try to take a look at this next week.

**Comment by pejuko** [comment]
Ok. Thanks. No hurry. I've solved this already in my second commit and it is working for me now. If this solution is ok for you, you can merge it or write me and I'll refactor.
